tests/update/patch_before_migration.pm

```diff
@@ -61,7 +61,7 @@ sub patching_sle {
             # Perform sync ahead of reboot to flush filesystem buffers that probably reduce time of sync during rebooting
             # In the case no need to always enlarge timeout of wait_boot
             assert_script_run 'sync', 600;
-            type_string "reboot\n";
+            type_string "reboot -f\n";
```

Comment:
probably you can add comment which explaining why actually you need to add this option.
I think we also don't want to propagate this option to all architectures because it have issues in one.